src/bindeps_conda.jl

```diff
@@ -1,5 +1,5 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
-using BinDeps
+import BinDeps
```

Comment:
Ok, got it. This would be a nice addition to define specialized version of `BinDeps.bindir` for the `Conda.Manager` type. This will be for another PR !